scripts/mkcloud

```diff
@@ -1095,6 +1095,21 @@ function crowbarupgrade_5plus()
     fi
 }
 
+function cloud6upgrade()
```

Comment:
what is the benefit of putting it in a separate function instead of adding another "elif" case to the generic "cloudupgrade" function (which was meant to do the upgrade no matter from which version)?